tests/Unit/Domain/PaymentUrlGenerator/NullGeneratorTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\PaymentContext\Tests\Unit\Domain\PaymentUrlGenerator;
+
+use PHPUnit\Framework\TestCase;
+use WMDE\Fundraising\PaymentContext\Domain\PaymentUrlGenerator\NullGenerator;
+
+/**
+ * @covers \WMDE\Fundraising\PaymentContext\Domain\PaymentUrlGenerator\NullGenerator
+ */
+class NullGeneratorTest extends TestCase {
+
+	public function testURLGenerationReturnsNull(): void {
+
+		//TODO test factory?
```

Comment:
This test should only test that the generator returns null, no need to create the generator with the factory.

The test for the factory should test that the configuration is passed to the generator constructor and that the right instance is returned for each payment type